pkg/actions/addon/get.go

```diff
@@ -99,6 +99,10 @@ func (a *Manager) findNewerVersions(addon *api.Addon) (string, error) {
 		return "", err
 	}
 
+	if len(versions.Addons) == 0 {
+		return "-", nil
+	}
+
```

Comment:
while running integration tests it was the first time we've not had a new version of vpc-cni available, so only just spotted this panic that needed fixing.